pyproject.toml

```diff
@@ -47,7 +47,6 @@ autohooks = ">=21.6.0"
 pontos = "^22.1.1"
 pytest = "*"
 pylint = "^2.12.2"
-isort = "^5.8.0"
```

Comment:
is this by intention?

Reply:
Yes. If `isort` is already a dependencies it doesn't need to be a dev dependencies, too! :)

Reply:
Ah just removed from the devs section. Should have looked at the whole file :-)